patch/files/scripts/items.gd

```diff
@@ -2059,30 +2059,82 @@ func sexchangepoteffect():
 
 #recipes
 func recipedecrypt(item):
+	#print("recipedecrypt")
+	craftitemingredients.clear()
 	var text = ''
 	var recipe = item.recipe
 	var canmake = true
 	for i in get(recipe):
-		var ingredient = globals.itemdict[i]
-		var amount = get(recipe)[i]
-		text += ingredient.name + ' - '+ str(amount) + ', '
-		if ingredient.amount < amount:
-			canmake = false
+		###---Added By Expansion ---###
+		if (globals.items.itemlist[i].type == "potion" || globals.items.itemlist[i].type == "ingredient"):
+			var ingredient = globals.itemdict[i]
```

Comment:
This line and a few others are shared with the else statement, recommend moving them above the if statement.